run-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+: "${NIXPKGS_CHANNEL:=nixpkgs-unstable}"
+
+nix_options=(
+  --no-out-link
+  #--option allow-import-from-derivation false
+  #--option restrict-eval true
+  --option sandbox true
+  --show-trace
+  -I "nixpkgs=channel:$NIXPKGS_CHANNEL"
+)
+
+nix-build tests "${nix_options[@]}"
```

Comment:
We probably want to explicitly set`--option allow-import-from-derivation true` here.

Reply:
Could you also add a comment explaining that this tests yarn with and without IFD?

Reply:
Good call, fixed in ffa06ac84a5158cf97609f6c5249cdf0dc8b36c1